android/lib/src/main/java/com/swmansion/gesturehandler/GestureHandlerOrchestrator.kt

```diff
@@ -245,17 +242,25 @@ class GestureHandlerOrchestrator(
     // approach when we want to use pointer coordinates to calculate velocity or distance
     // for pinch so I don't know yet if we should transform or not...
     event.setLocation(coords[0], coords[1])
-    handler.handle(event)
-    if (handler.isActive) {
-      handler.dispatchTouchEvent(event)
+    if (handler.needsPointerData) {
+      handler.updatePointerData(event)
     }
-    event.setLocation(oldX, oldY)
-    // if event was of type UP or POINTER_UP we request handler to stop tracking now that
-    // the event has been dispatched
-    if (action == MotionEvent.ACTION_UP || action == MotionEvent.ACTION_POINTER_UP) {
-      val pointerId = event.getPointerId(event.actionIndex)
-      handler.stopTrackingPointer(pointerId)
+
+    if (!handler.isAwaiting || action != MotionEvent.ACTION_MOVE) {
```

Comment:
This seem to be changing the event processing logic. What is the reason behind this change? We will no longer call `handle` in certain conditions and I wonder if it is not going to break something?

Reply:
I wanted to send pointer events event if the handler was awaiting the activation of another handler so I couldn't do it in the `handle` method because it's not called in that case. So I slightly modified `deliverEventToGestureHandler` to handle pointer events there. I don't really see the case where `handle` wouldn't be called now but was called before. If you were referring to [this line](https://github.com/software-mansion/react-native-gesture-handler/pull/1673/files/2334e124b1f2d5e02fbc8ca473aa338f1827c0b0#diff-0183690c13c9a7694f9bad1e3e57c96de12db967f2aa3d8a5c02b41b76ef9c88R249) then it's doing the same check as the if I have removed above, just later.